File: xv/patch/XV2020Global.swift

extension XV2020 {
  
  enum Global {
    
    static let patchWerk = XV.sysexWerk.multiPatchWerk("Global", [
      ([.common], 0x0000, XV5050.Global.Common.patchWerk),
    ], start: 0x02000000, initFile: "xv2020-global-init")
    
  }
  
}

File: xv/xv2020_global.js
const XV5050Global = require('./xv5050_global.js')

const patchWerk = {
  multi: 'Global',
  map: [
    ["common", 0x0000, XV5050Global.commonPatchWerk),
  ],
  initFile: "xv2020-global-init",
}

module.exports = {
  patchWerk,
}
